modules/rapidjson/1.1.0.bcr.20241007/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+  bazel: [6.x, 7.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
```

Comment:
You could add `parse_headers` and `process_headers_in_dependencies` for basic CI coverage

Reply:
good idea, i just fixed the tests instead